packages/core/editor/src/components/Editor/Editor.tsx

```diff
@@ -25,7 +25,6 @@ type Props = {
 };
 
 const getEditorStyles = (styles: CSSProperties) => ({
-  paddingBottom: 150,
   ...styles,
```

Comment:
Let's create new `style` prop for <YooptaEditor /> component with default values 
```
{
  width: 400,
  paddingBottom: 100
}
```

Reply:
For compatibility leave the prop "width", but it should be marked as the old prop.
So, default value should look like this

```
const editorStyles = {
  width: width || style.width || 400,
  paddingBottom: style.paddingBottom || 100
}
```